dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -783,6 +783,12 @@ void AnimatedVisualPlayer::OnAutoPlayPropertyChanged(
 void AnimatedVisualPlayer::OnAnimationOptimizationPropertyChanged(
     winrt::DependencyPropertyChangedEventArgs const& args)
 {
+    // We do not support animated visuals below RS5, so nothing to do.
```

Comment:
This comment isn't accurate, we do support animated visuals, but we don't support animation optimizations.

Reply:
I'd suggest just deleting the comment entirely, the code imparts this information already.

Reply:
I copied it from another place in this file. I can change it

Reply:
Removed